dz_s2/zadacha_2_dz.py

```diff
@@ -0,0 +1,45 @@
+# Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем
+# и знаменателем. Программа должна возвращать сумму и произведение* дробей.
+# Для проверки своего кода используйте модуль fractions.
+import fractions
+
+
+def reduce_fract(numerator, denominator):
+    cut = 1
+    for i in range(2, denominator):
+        if denominator % i == 0 and numerator% i == 0:
+            cut = i
+    return cut
+
+
+def prog(str_1, str_2):
+    str_x1, str_x2 = str_1.split('/')
+    str_y1, str_y2 = str_2.split('/')
+    x1 = int(str_x1)
+    x2 = int(str_x2)
+    y1 = int(str_y1)
+    y2 = int(str_y2)
```

Comment:
Можно сразу же перевести  `int`
```suggestion
    x1, x2 = map(int, str_1.split('/'))
    y1, y2 = map(int, str_2.split('/'))
```